lib/compat/wordpress-6.0/class-wp-webfonts.php

```diff
@@ -291,4 +300,24 @@ public function generate_styles() {
 
 		return $styles;
 	}
+
+	/**
+	 * Preload webfonts to improve performance
+	 *
+	 * @return void
+	 */
+	public function preload_webfonts() {
+		foreach ( $this->get_fonts() as $font ) {
+			if ( $font['preload'] ) {
```

Comment:
Should we add a check here to make sure there are no warnings in case `preload` is not defined?
```suggestion
			if ( ! empty( $font['preload'] ) && $font['preload'] ) {
```